testdata/testinput2

```diff
@@ -4220,6 +4220,37 @@
     apple lemon blackberry
 
 /abc/
+    123abc123\=replace=XYZ
+    123abc123\=replace=[10]XYZ
+\= Expect error
+    123abc123\=replace=[9]XYZ
+    123abc123\=substitute_overflow_length,replace=[9]XYZ
+    123abc123\=substitute_overflow_length,replace=[6]XYZ
+    123abc123\=substitute_overflow_length,replace=[1]XYZ
+    123abc123\=substitute_overflow_length,replace=[0]XYZ
```

Comment:
I'd be curious to run these new tests against the old code, just to see which (if any) of the test outputs have changed.

Reply:
None does, but could add split the tests in a "setup" patch of its own so it will be obvious